lib/multiplexer/multiplexer.go

```diff
@@ -180,6 +203,7 @@ func (m *Mux) Wait() {
 // and accepts requests. Every request is served in a separate goroutine
 func (m *Mux) Serve() error {
 	defer m.waitCancel()
+	go m.logLimiter.Run(m.context)
```

Comment:
Let's do this where the logLimiter is created, not here.

Reply:
Done in 3554daa6db947bd222191b45f922fa428c044673.